src/plugins/telemetry/server/routes/telemetry_config.ts

```diff
@@ -36,6 +36,9 @@ export function registerTelemetryConfigRoutes({
     {
       path: FetchTelemetryConfigRoute,
       validate: false,
+      options: {
+        authRequired: 'optional',
+      },
```

Comment:
this allows us to fetch configuration while on the login screen.

Reply:
I assume there is nothing sensitive in the response, right?

Reply:
The structure of the output is declared in https://github.com/elastic/kibana/blob/0fcc9e0a754a464b92aab456af186e9d26ec5b2a/src/plugins/telemetry/server/routes/telemetry_config.ts#L83-L89

And our awesome tests fail validation if any additional info is exposed (kudos to @jloleysens 🧡)